MeshLib/MeshSurfaceExtraction.h

```diff
@@ -91,13 +97,21 @@ class MeshSurfaceExtraction
         double angle,
         unsigned mesh_dimension);
 
-    /// Functionality needed for getSurfaceNodes() and getMeshSurface()
-    static void get2DSurfaceNodes(
-        std::vector<MeshLib::Node*>& sfc_nodes,
-        std::size_t n_all_nodes,
-        const std::vector<MeshLib::Element*>& sfc_elements,
-        std::vector<std::size_t>& node_id_map);
+    static std::tuple<std::vector<MeshLib::Node*>, std::vector<std::size_t>>
```

Comment:
Can this static member functions be free (functions)?

Reply:
Yes, most (maybe all) of the static member functions can be free. I don't want to break the 'usual' behavior in this file. I even think that class is unnecessary.